test/TestAccountHelper.js

```diff
@@ -0,0 +1,219 @@
+const nearAPI = require('near-api-js');
+
+const constants = require('../constants');
+
+const { SERVER_EVENTS } = constants;
+
+function extractValueFromHash(hash, key) {
+    const value = hash[key];
+    delete hash[key];
+
+    return value;
+}
+
+class TestAccountHelper {
+    constructor({
+        app,
+        ECHO_SECURITY_CODES = false,
+        keyPair,
+        keyStore,
+        request,
+    }) {
+        this._keyPair = keyPair;
+        this._keyStore = keyStore;
+        this._request = request;
+        this._app = app;
+
+        this._signer = new nearAPI.InMemorySigner(this._keyStore);
+        this._securityCodesByAccountId = {};
+
+        if (this._app) {
+            app.on(SERVER_EVENTS.SECURITY_CODE, ({ accountId, securityCode, requestId }) => {
+                if (ECHO_SECURITY_CODES) {
+                    console.log('Got security code', { accountId, securityCode, requestId });
+                }
+                this._securityCodesByAccountId[accountId] = securityCode;
+            });
+        }
+    }
+
+    async ensureNEARApiClientInitialized() {
+        if (!this._NEARApiClient) {
+            this._NEARApiClient = await nearAPI.connect({
+                deps: { keyStore: this._keyStore },
+                nodeUrl: process.env.NODE_URL
+            });
+        }
+    }
+
+    get near() {
```

Comment:
not sure I understand why define explicit getters for things where you can expose property directly